api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPersonServiceImpl.java

```diff
@@ -17,38 +17,42 @@
 import ca.uhn.fhir.rest.param.StringOrListParam;
 import ca.uhn.fhir.rest.param.TokenOrListParam;
 import lombok.AccessLevel;
+import lombok.Getter;
 import lombok.Setter;
 import org.hl7.fhir.r4.model.Person;
 import org.openmrs.module.fhir2.api.FhirPersonService;
+import org.openmrs.module.fhir2.api.dao.FhirDao;
 import org.openmrs.module.fhir2.api.dao.FhirPersonDao;
+import org.openmrs.module.fhir2.api.translators.OpenmrsFhirTranslator;
 import org.openmrs.module.fhir2.api.translators.PersonTranslator;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
 @Component
 @Transactional
+@Getter(AccessLevel.PROTECTED)
 @Setter(AccessLevel.PACKAGE)
-public class FhirPersonServiceImpl implements FhirPersonService {
+public class FhirPersonServiceImpl extends BaseFhirService<Person, org.openmrs.Person> implements FhirPersonService {
 	
 	@Autowired
-	private FhirPersonDao fhirPersonDao;
+	private FhirPersonDao Dao;
 	
 	@Autowired
-	private PersonTranslator personTranslator;
+	private PersonTranslator translator;
 	
 	@Override
 	@Transactional(readOnly = true)
-	public Person getPersonByUuid(String uuid) {
-		return personTranslator.toFhirResource(fhirPersonDao.getPersonByUuid(uuid));
+	public Person get(String uuid) {
+		return translator.toFhirResource(Dao.get(uuid));
 	}
```

Comment:
You should be able to remove this function altogether

Reply:
Finished